xoinvader/ingame.py

```diff
@@ -251,9 +249,9 @@ def events(self):
 
     def update(self):
         self._collision_manager.update()
-        self.wave.update()
-        if not self.wave.running:
-            self.wave.start()
+        self.level.update()
```

Comment:
Maybe move this code to level's update like I did with animation? And here just add level to objects like any other GameObjects.